### DISCUSSION TITLE
What issue template for formatting fix?

### DISCUSSION THREAD

**Discussion by madsgillis** [nvaccess/nvda#16605] [Q&A]
I need to open an issue for a class I'm taking and I want to start very, very small. I found a file that isn't correctly formatted and would like to open that as an issue and then make a pull request for it. Is that acceptable?

If it is, what template should I use to report the issue? I feel like it doesn't fit into bug report or feature request, but it also doesn't seem big enough to use the special case issue template. Thanks!

**Comment by hwf1324** [comment]
Very welcome and thank you for your intention to contribute to NVDA.

In the "[Code/Docs contributions](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/contributing.md#guidelines)' document:

> A minor/trivial change which definitely wouldn't require design, user experience or implementation discussion, you can just create a pull request rather than using an issue first.

I guess if it's a very clear formatting error then opening a PR will do the trick.

**Reply by madsgillis** [reply]
Gotcha. Thank you for your answer!

**Comment by josephsl** [comment]
Hi, in the upcoming pull request, please describe the formatting issue(s) you found in the file in question so reviewers can discuss it. Thanks (and hope this would be a great learning experience inside and outside the classroom).

**Reply by madsgillis** [reply]
Will do, thanks!
